src/Sarif.FunctionalTests/SarifConverterTestData/AndroidStudio/AndroidStudio_issueLog1_raw.xml.sarif

```diff
@@ -17,6 +17,36 @@
           }
         ]
       },
+      "logicalLocations": {
+        "app\\microsoft.androidstudioformatexample\\microsoft.androidstudioformatexample.ApplicationTest ApplicationTest()": [
+          {
+            "name": "app",
+            "kind": "android-module"
```

Comment:
I think we need `function` for languages that have free functions, and `type` for languages with kinds of types other than the ones we mention (and also because in FxCop, your `context` object just has a `type` member, and doesn't specify what kind of type it is).